extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -23,4 +23,5 @@ amp-sticky-ad {
   width: 100%;
   /* TODO(zhouyx, #3250): discuss on what the max-height should be? */
   max-height: 100px !important;
+  font: 0/0 a; /* remove the gap between inline(-block) elements */
```

Comment:
When I put `amp-ad` inside `amp-sticky-ad`, The `amp-sticky-ad` height will expand a few pixels. I think it has something to do with `amp-ad` display type inline block. Changing the font to 0 solves the problem.